x-pack/plugins/infra/public/pages/metrics/hosts/components/kpi_charts/kpi_chart.tsx

```diff
@@ -122,4 +122,10 @@ const KPIChartStyled = styled(Chart)`
   .echMetric {
     border-radius: ${(p) => p.theme.eui.euiBorderRadius};
   }
+  button {
+    cursor: default;
+  }
+  div {
+    pointer-events: none;
+  }
```

Comment:
Can we use the `.echMetric` selector to set the `pointer-events` in this case? So something like: 
```suggestion
  .echMetric {
    border-radius: ${(p) => p.theme.eui.euiBorderRadius};
    pointer-events: none;
  }
```

Reply:
Yes I guess this should also work and is better, thanks for the suggestion 👍